### ISSUE TITLE
Remove `forc index start`

### ISSUE THREAD

**Issue by ra0x3** [FuelLabs/fuel-indexer#497]
- `forc index start` was originally added so that _all_ functionality could go through the plugin - at a time when we didn't know that most users would just use binaries made available in `fuelup`
- However, `forc index start` is now no different than just calling the binary directly `./fuel-indexer --whatever` (it just wraps starting the binary manually)
- `forc index start` seems to be an unnecessary layer and might confuse users
  - Since all `fuel-indexer` binary use is via `fuelup` now, `forc index start` is overly repetitive and should be removed 

Additional context
- We don't reference `forc index start` in the new [Quickstart](https://fuellabs.github.io/fuel-indexer/master/quickstart/index.html)
- We would need to remove references to `forc index start` from the new README https://github.com/FuelLabs/fuel-indexer/issues/490, and the new book https://github.com/FuelLabs/fuel-indexer/issues/489
- This could be seen as a breaking change if there are users relying `forc index start` (I don't believe this to be the case, but could be)

**Comment by deekerno** [comment]
I think that this is a good change.

If we remove `forc index start`, I think that we should couple that with the addition of a `run` argument to the `fuel-indexer` binary, as that's what is done with `fuel-core` to run a node; I think it's best to keep in line with the rest of the Fuel ecosystem regarding command invocations.

**Comment by ra0x3** [comment]
@deekerno 
- I see [`fuel-core run`](https://github.com/FuelLabs/fuel-core/blob/master/bin/fuel-core/src/cli.rs#L92)
- So you're saying make the `fuel-indexer` binary use a set of subcommands? (as opposed to a single set of CLI opts now)

**Comment by deekerno** [comment]
Yeah, I think that would be best. You can run the Fuel node by either invoking `fuel-core` or `fuel-core run`, so I think we should mirror that as best we can (i.e. `fuel-indexer` and `fuel-indexer run`) to reduce mental load between different Fuel components.

**Comment by ra0x3** [comment]
Ahh so it’s additive (not breaking), got it. Sounds good to me, I’ll create a separate issue for that.

**Comment by rvmelkonian** [comment]
are these `fuel-indexer` commands merged and ready to go? Should I update the docs to use these commands and not `fuelup`?

**Comment by deekerno** [comment]
> are these `fuel-indexer` commands merged and ready to go? Should I update the docs to use these commands and not `fuelup`?

Not just yet, #551 should be reviewed first; otherwise, users will have no way to start the service. 😅

**Comment by ra0x3** [comment]
@deekerno @rvmelkonian Are we saying this is blocked by #551? In what way?

**Comment by deekerno** [comment]
@ra0x3 Oh, you're right, it's definitely not; I looked at the comment too quickly and replied. My apologies.

**Comment by rvmelkonian** [comment]
Ok, I'll update those docs then.

**Comment by ra0x3** [comment]
- So  new feedback I've gotten on this is that `forc index start` should remain, as it's more ergonomic than interacting with the binaries directly. 
- However, if we're gonna keep it, we should update the Quickstart docs to use `forc index start` instead of calling the binary directly. I can do this in https://github.com/FuelLabs/fuel-indexer/pull/558
- Lmk your thoughts @rvmelkonian and @deekerno

**Comment by deekerno** [comment]
That's fine by me. If it stays, then we can probably close #551 as there would be no need to conform to the convention used by `fuel-core`.

**Comment by rvmelkonian** [comment]
Ok. If it's going in #558 then we should close this issue?

**Comment by deekerno** [comment]
Yeah, I'll close this issue as we've got all the documentation bits covered in other PRs.
